app/views/layouts/partners/application.html.erb

```diff
@@ -6,6 +6,8 @@
   <title>PartnerBase</title>
   <meta content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" name="viewport">
   <%= csrf_meta_tags %>
+  <% # It is important to load the styling before AdminLTE via stylesheet_link_tag 'application' %>
+  <link href="https://cdn.jsdelivr.net/npm/select2@4.1.0-rc.0/dist/css/select2.min.css" rel="stylesheet" />
```

Comment:
The remove-webpacker PR might affect this...

Reply:
Not offhand - but I can see I added this in separately in that commit: https://github.com/rubyforgood/human-essentials/pull/3209/files#diff-635bf6f5bc607736c2b00fa13a1f31c3aa030fe7719df80f88bcd98b813075a6R12